docs/parachain_onboarding.md

```diff
@@ -0,0 +1,73 @@
+# Parachain Onboarding to Paseo Testnet Wiki
+
+## Introduction
+
+Welcome to the Paseo Testnet onboarding guide for parachains. This document provides a comprehensive guide for projects looking to join the Paseo Testnet as a parachain. Here you'll find step-by-step instructions, required configurations, and best practices to ensure your integration is successful.
+
+## Table of Contents
+
+1. **Prerequisites**
+2. **Parachain Registration Process**
+3. **Setting Up Your Parachain**
+4. **Connecting Your Parachain to Paseo**
+5. **Monitoring and Maintenance**
+6. **FAQ**
+7. **Support and Resources**
+
+## 1. Prerequisites
+
+Before initiating the onboarding process, ensure you meet the following requirements:
+
+- **Technical Requirements:** Have a fully operational blockchain capable of connecting as a parachain. This includes having a collator setup ready.
+- **Resource Allocation:** Ensure you have the computational and network resources to run a stable collator node.
+- **Paseo Tokens:** Acquire sufficient PAS tokens for staking if required for becoming a parachain on the Paseo Testnet.
+
+## 2. Parachain Registration Process
+
+To register your blockchain as a parachain on the Paseo Testnet:
+
+### 2.1 Application Submission
+- Submit an application through the official Paseo Testnet website. Include technical details of your blockchain, team information, and your objectives for joining the testnet.
```

Comment:
Right now we don't have a website live.
Maybe better point people to open an issue in the support repo.

Reply:
updated! can you please check it again?